articles/django-rest-api-mongodb-notes-app/django-rest-api-mongodb-notes-app.md

```diff
@@ -0,0 +1,225 @@
+# Build a Notes App API with Django REST Framework and MongoDB
+
+## Introduction
+
+MongoDB is a NoSQL database system. It stores data in JSON-like format instead of the traditional row-column format. The Django REST framework is a powerful toolkit for building extensible APIs. The Django framework is a Python framework for web development.
+
+You will learn how to use the powerful Django REST framework and MongoDB in your web projects. You will learn how to put in place CRUD features in a notes app.
+
+In this post, you will:
+
+- Understand to integrate MongoDB into Django projects
+- Learn how to set up and run Django REST API
+- Create CRUD features in Django REST API
+- Build a Notes app API
```

Comment:
You have stated that you are to "Create CRUD features ..." while in this tutorial you have looked at Create and Get only. Is this article complete? If yes, please edit this statement to reflect what you have looked at in this tutorial.

Reply:
Thanks for the review @geoffrey45 

Update and Delete are possible through the Get endpoint (i.e NoteDetail). Maybe I should add an image to illustrate them.

Reply:
- I have added the image to illustrate Delete and Update. The illustrations for Create and Retrieve were already in the article.
- I have also added the syntax highlights to the code snippets.
Thanks.